bindings/rust/wasmedge-sys/src/types.rs

```diff
@@ -278,20 +278,29 @@ pub enum Value {
     ///
     /// The packed data can be interpreted as signed or unsigned integers, single or double precision floating-point
     /// values, or a single 128 bit type. The interpretation is determined by individual operations.
-    V128(u128),
+    V128(i128),
     /// A reference to [functions](crate::Function).
-    FuncRef(u128),
+    FuncRef(u32),
     /// A reference to object owned by the [Vm](crate::Vm).
     ExternRef(u128),
+    /// A null reference value.
+    NullRef(RefType),
 }
 impl Value {
-    pub fn gen_extern_ref(&mut self) -> Value {
+    pub fn gen_extern_ref<T>(extern_obj: &mut T) -> Value {
         unsafe {
-            let self_ptr: *mut c_void = self as *mut _ as *mut c_void;
-            wasmedge::WasmEdge_ValueGenExternRef(self_ptr).into()
+            let ptr = extern_obj as *mut T as *mut c_void;
+            wasmedge::WasmEdge_ValueGenExternRef(ptr).into()
         }
     }
 
+    // pub fn gen_extern_ref(&mut self) -> Value {
+    //     unsafe {
+    //         let self_ptr: *mut c_void = self as *mut _ as *mut c_void;
+    //         wasmedge::WasmEdge_ValueGenExternRef(self_ptr).into()
+    //     }
+    // }
```

Comment:
Why do we need to keep these comments?

Reply:
The function has been replaced by a new one. I'll remove these lines. Thanks!

Reply:
The issue has been fixed, please check again. Thanks!